bve-render/src/render.rs

```diff
@@ -130,6 +130,7 @@ pub fn create_pipeline(
     fs: &ShaderModule,
     samples: MSAASetting,
 ) -> RenderPipeline {
+    debug!("Creating opaque pipeline: samples: {:?}", samples);
```

Comment:
Protip: log the name of the pipeline you're creating

Reply:
I did, `opaque`

Reply:
oh